apps/desktop-wallet/announcement.json

```diff
@@ -0,0 +1,9 @@
+{
+  "isActive": true,
+  "title": "This is the first announcement!",
+  "description": "Implemented with a very elegant solution, that will make Mikapuccino very happy! Let's see!",
+  "button": {
+    "title": "Download",
+    "link": "https://github.com/alephium/alephium-frontend/"
+  }
```

Comment:
Needs to be updated before we merge this. Let's keep it like this for testing now.